Microsoft.Identity.Web/ConfidentialClientApplicationOptionsWithCertificate.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Identity.Client;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Extension of ConfidentialClientApplicationOptionsWithClientCertificate with information
+    /// about the Client certificate
+    /// </summary>
+    public class ConfidentialClientApplicationOptionsWithClientCertificate : ConfidentialClientApplicationOptions
```

Comment:
>ConfidentialClientApplicationOptionsWithClientCertificate [](start = 17, length = 57)

Shouldn't this ideally be added as a property in ConfidentialClientApplicationOptions class itself?